content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,278 @@
+
+
+- The ability to more granularly manage the software dependencies available to Edge clusters. 
+
+
+This how-to guide provides instructions for creating and using a content bundle to build the Edge artifacts. You will begin with installing a necessary tool, the Palette Edge CLI, on your development machine. The Palette Edge CLI is a command-line utility to interact with Palette and perform specific tasks in your development environment, such as creating a content bundle. Next, you will download all the software dependencies mentioned in your cluster profile using the Palette Edge CLI and create a content bundle. Lastly, when your content bundle is ready, you will use the CanvOS utility to embed the content bundle and user data into the Edge installer ISO image.    
+
+The diagram below displays the overarching steps to build the Edge installer ISO using a content bundle. The diagram also highlights the primary prerequisites to create a content bundle.
+
+![An overarching diagram displaying the workflow in the current guide.](/clusters_edge_edge-forge-workflow_build-images_build-artifacts_overarching.png)
+
+
+# Prerequisites
+
+<WarningBox>
+
+  This how-to guide extends the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) workflow. Therefore, you must complete it before proceeding with the current guide. 
+
+</WarningBox>
+  
+
+To complete this guide, you will need the following items:
+<br />
+
+* A physical or virtual Linux machine with *AMD64* (also known as *x86_64*) processor architecture to build the Edge installer ISO image. You can issue the following command in the terminal to check your processor architecture. 
+  <br/>
+
+  ```bash
+  uname -m
+  ```
+
+* The Linux machine should have the following minimum hardware configuration:
+  - 4 CPU
+  - 8 GB memory
+  - 100 GB storage. The actual storage will depend on the size of the content bundle you will use to build the Edge installer ISO image.
+
+
+* You must have completed the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) guide to build the provider images and create a cluster profile referencing one of the provider images. 
+
+
+* A Spectro Cloud API key. Later in this guide, you will use this API key to authenticate the Palette Edge CLI utility and allow it to interact with the Palette. Refer to the [User Authentication](https://docs.spectrocloud.com/user-management/user-authentication/#apikey) guide to create a new API key. 
+
+
+# Instructions
+
+Use the following instructions on your Linux machine, which this guide refers to as the development environment. 
+<br />
+
+1.  Visit the [Downloads](https://docs.spectrocloud.com/spectro-downloads#paletteedgecli) page and download the latest Palette Edge CLI. You can download the Palette Edge CLI by clicking on the available URL or using the download URL in the following command. Replace the `[PALETTE-EDGE-BINARY-URL]` placeholder with the download URL. 
```

Comment:
I think it might be easier to tell the user to download using the browser, they can right-hand click on the URL or just click on it. Most browsers will start the download after clicking on the URL. 

If we do that, then we can remove the `curl` step. And go straight to step 2.